components/library/libxml2/history

```diff
@@ -1,3 +1,3 @@
-library/python/libxml2-34@2.9.9,2020.0.1.1
+library/python/libxml2-34@2.9.9-2020.0.1.1
```

Comment:
The versions were missing "5.11-".

Reply:
There are no 5.11- in the FMRI:
pkg info libxml2-34
...
FMRI: pkg://openindiana.org/library/python/libxml2-34@2.9.9-2020.0.1.0:20200330T151116Z
...

Reply:
There should be since FMRIs have versions @$(IPS_COMPONENT_VERSION),$(BUILD_VERSION) and BUILD_VERSION is defined as
https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/make-rules/ips-buildinfo.mk#L62

If not, then basically we have inconsistent FMRIs across oi-userland.
I do not know what kind of magic makes the ,5.11 disappear in the repository's FMRIs but since I had the FMRIs generated from the pkg repo, that definitely explains why they were all wrong.

Reply:
Arbitrary checks:
FMRI: pkg://openindiana.org/shell/zsh@5.7.1-2020.0.1.0:20200330T161639Z
FMRI: pkg://openindiana.org/shell/bash@5.0.17-2020.0.1.5:20200525T181710Z
FMRI: pkg://openindiana.org/developer/gcc-10@10.1.0-2020.0.1.1:20200526T093716Z
FMRI: pkg://openindiana.org/runtime/nodejs-14@14.5.0-2020.0.1.0:20200701T171326Z
We seem to have lots of package without "5.11-" in userland.

But packages from illiumos-gate have it:
FMRI: pkg://openindiana.org/driver/storage/mpt_sas@0.5.11-2020.0.1.19909:20200713T015035Z
FMRI: pkg://openindiana.org/system/library/libutempter@0.5.11-2020.0.1.19909:20200713T023904Z

Reply:
Don't mess up version and build version
Look at 
set name=pkg.fmri value=pkg://userland/package/pkg@0.5.11,5.11-2020.0.1.5538:20200516T063327Z
Here 0.5.11 is version and 5.11 is build release (platform, on which package was built). 
It's set to 5.11 by default by oi-userland transforms. 
Once pkg changed its behavior - https://github.com/OpenIndiana/pkg5/commit/70a649f9f2f9702950b250e1add20d2732df6b94 - and avoids printing long versions (containing packaging date and build release).